src/scripts/downloadUiAssets.js

```diff
@@ -322,12 +403,28 @@ async function main() {
   console.log("Downloading from", url);
 
   try {
-    // Check versions compatibility
-    await exports.checkCompatibility(await getVersions(url), url);
-    // Get Workspace Web Edition archive
-    await getArchive(url + 'ui/wwe/', './ui-assets/wwe');
-    // Get GWS Auth UI archive
-    await getArchive(url + 'ui/auth/', './ui-assets/auth');
+    let isAzurePlatform;   
+    isAzurePlatform = await checkUrls(url);
+    if(isAzurePlatform === 'retry') {
+      isAzurePlatform = await checkUrls(url, 'ui/wwe/');
+    }
+    if (isAzurePlatform === true) {
+      authUrl = await getGwsApiUri(url)+'/ui/auth/';
```

Comment:
Isn't guaranty that in the future Auth UI will be  on the same domain than GWS.
Why did you not try to get the information from GWS like mentioned in the Jira ticket?

Reply:
Fixed